### DISCUSSION TITLE
Archives (.pak) greyed out

### DISCUSSION THREAD

**Discussion by xHybred** [4sval/FModel#448] [Q&A] [answered]
I tried the latest FModel and FModel QA Testing builds, this file is still greyed out no matter what.

I'm an UE5 developer and want to see what Epic Games did to fix certaim UE5 issues so I want to accessDefaultEngine.ini DefaultScalability BaseEngine.ini BaseScalability & WindowsEngine.ini

None of these are large files, I'm not trying to asset rip or debug the game in any way, just some basic UE5 cvars and whatnot. Is there any way I can get access to these specific files? Thanks for the response

**Comment by NotOfficer** [comment, marked as answer]
You might be using the wrong AES key.

**Reply by xHybred** [reply]
What is the correct AES key for that file? Because I was unable to retrieve it following traditional methods like AES finder and I thought FModel came with keys inside of it anyways?

**Reply by NotOfficer** [reply]
You can find the latest key [here](https://fortnitecentral.genxgames.gg/api/v1/aes).

**Reply by xHybred** [reply]
I appreciate it however none of those keys pertain to Pakchunk0-Windows.pak, the main key for example is already entered into FModel and the dynamic keys seem to be fore different Pakchunks based on their name.

Unless you're seeing something I'm not, I don't believe any of those keys help sadly. 

Also I took a look at Archive info, here's what it says

    "Name": "pakchunk0-WindowsClient.pak",
    "Length": 683692340,
    "FileCount": 0,
    "MountPoint": null,
    "IsEncrypted": true,
    "IsEnabled": false,
    "Key": "",
    "Guid": "00000000-00000000-00000000-00000000"

**Reply by 4sval** [reply]
As shown in the JSON, these are keys for the latest version of the game. If they don't work, you don't have the latest version or you're not telling everything.

**Reply by xHybred** [reply]
Fortnite updates so often, I just played it not that long ago but it might already have another update out that I was unaware of. |

Because of which I'll check and get back to you if it did and if the update fixed it. Cya soon and thanks

**Reply by xHybred** [reply]
Thank you, that worked, my Fortnite wasn't up to date. I lplayed it not long ago so I'm surprised it happened. Thank you for your help, I closed the issue

**Reply by hamidfathi66** [reply]
> You can find the latest key [here](https://fortnitecentral.genxgames.gg/api/v1/aes).

How can keys be used in the FModel?
